website/content/docs/nia/license.mdx

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+page_title: Consul-Terraform-Sync Enterprise License
+description: >-
+  Consul-Terraform-Sync Enterprise License
+---
+
+# Consul-Terraform-Sync Enterprise License
+<EnterpriseAlert>
+  Licenses are only required for Consul-Terraform-Sync Enterprise
+</EnterpriseAlert>
+
+Consul-Terraform-Sync Enterprise binaries require a license in order to start. The required license must be a [Consul Enterprise license](/docs/enterprise/license/overview) as there is no specific Consul-Terraform-Sync Enterprise license. As a result, Consul-Terraform-Sync Enterprise's licensing is very similar to Consul Enterprise. Consul-Terraform-Sync Enterprise is available with all its features with any valid, unexpired Consul Enterprise license regardless of its packaging/pricing model or level.
+
+To get a trial license for Consul-Terraform-Sync Enterprise, you can sign-up for the [trial license for Consul Enterprise](/docs/enterprise/license/faq#q-where-can-users-get-a-trial-license-for-consul-enterprise).
+
+## Setting the License
+
+Consul-Terraform-Sync Enterprise binary is only offered without a built-in license. Depending on how you would like to set the license, have the license value at hand or saved in a file to disk.
+
+The options to set the license:
+ - `CONSUL_LICENSE`: environment variable to set the license value
+ - `CONSUL_LICENSE_PATH`: environment variable to set the path to the file containing the license
+ - [`license_path`](/docs/nia/configuration#license_path): configuration file variable to set the path to the file containing the license
+
+If multiple license options are set, the order of precedence is as follows:
+1. `CONSUL_LICENSE`
+1. `CONSUL_LICENSE_PATH`
+1. `license_path`
```

Comment:
You can condense the two bullet/numbered list you move where the "order of precedence" is mentioned!

```suggestion
The options to set the license in order of precedence:
 - `CONSUL_LICENSE`: environment variable to set the license value
 - `CONSUL_LICENSE_PATH`: environment variable to set the path to the file containing the license
 - [`license_path`](/docs/nia/configuration#license_path): configuration file variable to set the path to the file containing the license
```

Reply:
I was thinking that splitting the information would be easier to skim but don't think the redundancy is worth it. Thanks!